src/components/pages/AuthPages/Signup.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import PageLayout from '<templates>/PageLayout/PageLayout';
+import AuthForm from '<organisms>/AuthForm/AuthForm';
+import withCheckAuth from '<HOC>/withCheckAuth';
+import { SignUpData, SignUpRules } from './signupItems';
+import FlexContainer from
+  '<atoms>/layouts/FlexContainer/FlexContainer';
+import Image from '<atoms>/Image/Image';
+import ImageUrl from '<image>/home.png';
+import { signup } from './navItems';
+import headerMapper from '<helpers>/headerMapper';
+
+const navItems = headerMapper(signup);
+
+const SignUp = (props) => {
+  return (
+    <PageLayout navItems={navItems} >
+      <FlexContainer paddingTop='zero'>
+        <FlexContainer flexDirection='row' height='80%'>
```

Comment:
i think you should use the already existing image component since it is being reused.